clang/include/clang/Basic/AttrDocs.td

```diff
@@ -8087,3 +8087,10 @@ requirement:
   }
   }];
 }
+
+def ClspvLibclcBuiltinDoc : Documentation {
+  let Category = DocCatFunction;
+  let Content = [{
+Attribute used by clspv to identified functions coming from libclc.
```

Comment:
I think this needs quite a bit more details, please see the rest of the AttrDocs examples, they typically go into examples, and at least specify what things like clspv and libclc would mean.